src/Components/ActivationKeys/blank-state.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Title,
+} from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { AddCircleOIcon } from '@patternfly/react-icons';
```

Comment:
Do you think it's better to be more specifc with just this one icon? I'm not sure if calling this entire folder brings in extra data to the page.
`import { AddCircleOIcon } from '@patternfly/react-icons/dist/js/icons/add-circle-o-icon';`

Reply:
@jmbuscemi I'm not sure if it brings any extra data to the page, I assume it'll we be optimized during build process, and it's also safer to use relative path, bc this way any changes to patternfly file structure won't affect us